lab_3_ann_retriever/main.py

```diff
@@ -183,6 +239,17 @@ def vector2tokens(self, vector: Vector) -> list[str] | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not vector or not isinstance(vector, tuple) \
+                or not all(isinstance(elem, float) for elem in vector) \
+                or len(vector) != len(self._token2ind):
+            return None
+        returned_tokens = []
+        for i, value in enumerate(vector):
```

Comment:
Do flat

Reply:
i tried, i hope i understood what you meant